org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/panel/InfoPanel.java

```diff
@@ -3007,6 +3022,21 @@ public void sort(Comparator<Object> cmpr, boolean ascending) {
 		renderItems();
 	}
 
+	/**
+	 * Get table name from AD_Ref_Table of Column Name
+	 * @param refValID
+	 * @param columnName
+	 * @return MTable[] tables
+	 */
+	private MTable[] getTables(int refValID, String columnName) {
+		if(refValID > 0) {
+			return new MTable[] {MTable.get(Env.getCtx(), MRefTable.get(Env.getCtx(), refValID).getAD_Table_ID())};
+		}
+		else {
+			return MTable.getByKeyColumns(Env.getCtx(), new String[] {columnName}, null);
```

Comment:
This should be unnecessary. When AD_Reference_Value_ID is 0, you just need to remove the _ID part to get table name.

Reply:
ok, removed the unnecessary code